### PR TITLE
 Adding `flytekitplugins` from PyPI (`group-1`)

### PR DESCRIPTION
**Pull request by sugatoray**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sugatoray** [comment]
## Notes

Previously this was being worked in this PR.

- https://github.com/conda-forge/staged-recipes/pull/19644

The branch name was changed to `flytekitplugins-group1` and that closed the previous PR.

**Comment by sugatoray** [comment]
| SL# | Package Name | PyPI | Recipe Version | Notes |
|:---|:---|:---:|:---:|:---|
| 0. | `flytekitplugins-athena` | [PyPI][#pypi-package-athena] | `v1.0.5` | --- |
| 1. | `flytekitplugins-awsbatch` | [PyPI][#pypi-package-awsbatch] | `v1.0.5` | --- |
| 2. | `flytekitplugins-data-fsspec` | [PyPI][#pypi-package-data-fsspec] | `v1.0.5` | --- |
| 3. | `flytekitplugins-modin` | [PyPI][#pypi-package-modin] | `v1.0.5` | --- |
| 4. | `flytekitplugins-pandera` | [PyPI][#pypi-package-pandera] | `v1.0.5` | --- |
| 5. | `flytekitplugins-spark` | [PyPI][#pypi-package-spark] | `v1.0.5` | --- |
| 6. | `flytekitplugins-sqlalchemy` | [PyPI][#pypi-package-sqlalchemy] | `v1.0.5` | --- |

[#pypi-package-athena]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-awsbatch]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-data-fsspec]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-modin]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-pandera]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-spark]: https://pypi.org/project/flytekitplugins-sqlalchemy/
[#pypi-package-sqlalchemy]: https://pypi.org/project/flytekitplugins-sqlalchemy/

**Comment by sugatoray** [comment]
PR is ready. Request for review/merging.

cc: @conda-forge/help-python

**Comment by sugatoray** [comment]
@ocefpaf This might also require a restart (osx error). Since, you knew what you did last time, I am leaving a note for you here.

**Comment by sugatoray** [comment]
@BastianZim Thanks for the restart. A question for you: would it be possible to create a separate conda-forge team by the name `flytekitplugins`? This will help maintain all flytekitplugins via a single team.

**Comment by BastianZim** [comment]
Could be done but would need to be done by core. The more easy way is to have one feedstock and then use that feedstock team for all other feedstocks.

**Comment by sugatoray** [comment]
@BastianZim I believe the PR is ready now.

Also, I updated the Conda-Forger App recently to allow generating recipes from PyPI (with/without version) and GitHub (with/without release tag). A lot of what I learned is while working with you. So, thank you for always being so kind and professional.

:package: Conda-Forger App: https://tinyurl.com/conda-forger

:tada: Tweet: https://twitter.com/sugatoray/status/1552721366146863106

:sparkles: YouTube Video: https://youtu.be/9JaFqhYfIdo

**Comment by BastianZim** [comment]
My pleasure! :)

Great project by the way!

**Comment by BastianZim** [comment]
I'm on my phone, I'll review it later of no one else is around.

**Comment by sugatoray** [comment]
> I'm on my phone, I'll review it later of no one else is around.

No rush. Take your time. Thank you.

**Comment by BastianZim** [comment]
I haven't fully reviewed it yet but just a couple of comments:

1. You need to include the notice, if I understand the repo correctly: https://github.com/flyteorg/flytekit/blob/master/NOTICE
2. Instead of uploading the file you can also link it dynamically. That way you don't need to copy/paste it for every package and update it. Something like this should work: https://github.com/conda-forge/azure-quantum-feedstock/blob/bcafc05d082a41e4121acd06b7254f95542bcf09/recipe/meta.yaml#L8-L12 
3. For the team: I don't think that you can have an empty feedstock. But you can manually set the name of the feedstock. So you can have one under that name and then use that one. Maybe there is some sort of master plugin where this would fit? You can just add `feedstock-name: ...` under `extra`.

**Comment by sugatoray** [comment]
But flytekit-feedstock already includes that NOTICE.

- https://github.com/conda-forge/flytekit-feedstock/blob/main/recipe/NOTICE

Is it still necessary?

**Comment by BastianZim** [comment]
Yes because it's a different feedstock. The full license needs to be included in each distribution.

This feedstock seems like a good candidate for the group you were talking about?

**Comment by sugatoray** [comment]
I will reconsider the group later. For now am going to add the notice to these recipes and then finish the work on this group1 plug-ins. 

I would write some instruction for the Flyte ecosystem users on how to create a conda-forge recipe for flytekitplugins. This will help other users in future to add other plugins easily.

**Comment by sugatoray** [comment]
Done (added `NOTICE` file). As soon as the CI's finish running, this PR should be ready.

cc: @BastianZim

File: recipes/flytekitplugins-athena/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - python >=3.7
+  run:
+    - python >=3.7
+    - flytekit >=1.0.0b3,<1.2.0
+
+test:
+  imports:
+    - flytekitplugins.athena
+  # commands:
+  #   - pip check
```
Comment (BastianZim) [inline recipes/flytekitplugins-athena/meta.yaml:30]: Can you just quickly say why you commented this out? Then this should be ready to go.

Comment (sugatoray) [reply recipes/flytekitplugins-athena/meta.yaml:30]: The pip checks are commented out only when after the builds pass but the pip-checks were failing.

Comment (BastianZim) [reply recipes/flytekitplugins-athena/meta.yaml:30]: Ok but why were they commented out? Is that something that cannot be fixed?

Comment (sugatoray) [reply recipes/flytekitplugins-athena/meta.yaml:30]: As far as I recall, that was the case. It was complaining about not finding the same package that was already installed.

Comment (BastianZim) [reply recipes/flytekitplugins-athena/meta.yaml:30]: Ok strange. Maybe investigate this in the feedstock?

**Pull request merged**